src/components/home/Home.jsx

```diff
@@ -2,13 +2,15 @@ import React from "react";
 import LoginForm from "../auth/login/Login";
 import SignUpForm from "../auth/signUp/SignUp";
 import LanguageDropdown from "./LanguageDropdown";
+import Navbar from "../layout/nav/Navbar";
 
 export default function Home() {
   return (
-    <div>
+    <>
+      <Navbar />
```

Comment:
just a note that you dont have to do now: probably you will want a Layout of some sorts because later you will want login, signup, language dropdown, navbar on ALL your pages, not just the homepage

Reply:
You're right, I'm gonna move those to `App` or `Navbar` to be seen in all pages.